recipes/libcurl/all/conanfile.py

```diff
@@ -347,7 +347,7 @@ def _patch_mingw_files(self):
     def _build_with_autotools(self):
         with tools.chdir(self._source_subfolder):
             # autoreconf
-            self.run("./buildconf", win_bash=tools.os_info.is_windows, run_environment=True)
+            self.run("{} -fiv".format(tools.get_env("AUTORECONF")), win_bash=tools.os_info.is_windows, run_environment=True)
```

Comment:
```suggestion
            self.run("./buildconf", win_bash=tools.os_info.is_windows, run_environment=True)
```
to hit the button tomorrow morning, if we decide